src/main/scala/ai/privado/audit/DataElementDiscovery.scala

```diff
@@ -627,46 +630,52 @@ object DataElementDiscoveryJS {
             }
 
             case (param: MethodParameterIn) => {
-              if (ruleMemberInfo.contains(param.name)) {
-                workbookResult += List(
-                  key.fullName,
-                  key.file.head.name,
-                  getFileScoreJS(key.file.name.headOption.getOrElse(Constants.EMPTY), xtocpg),
-                  param.name,
-                  param.typeFullName,
-                  AuditReportConstants.AUDIT_CHECKED_VALUE,
-                  ruleMemberInfo.getOrElse(param.name, "Default value"),
-                  AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
-                  AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
-                  AuditReportConstants.AUDIT_EMPTY_CELL_VALUE
-                )
-              } else {
-                workbookResult += List(
-                  key.fullName,
-                  key.file.head.name,
-                  getFileScoreJS(key.file.name.headOption.getOrElse(Constants.EMPTY), xtocpg),
-                  param.name,
-                  param.typeFullName,
-                  AuditReportConstants.AUDIT_NOT_CHECKED_VALUE,
-                  AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
-                  AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
-                  AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
-                  AuditReportConstants.AUDIT_EMPTY_CELL_VALUE
-                )
+              val paramUniqueKey = s"${key.fullName}${key.file.head.name}${param.name}"
```

Comment:
update `key.file.head.name` to `key.file.name.headOption.getOrElse(Constants.EMPTY)`, 

Similarly update for all other occurrences